src/controllers/StreamController.js

```diff
@@ -0,0 +1,62 @@
+const Stream = require('../models').Stream;
+
+module.exports = {
+  async getAll(req, res) {
+    try {
+      const stream = await Stream.findAll();
+      return res.status(200).send(stream);
+    } catch (error) {
+      return res.status(500).send(error);
+    }
+  },
+  async addNew(req, res) {
+    const newStream = req.body;
+    try {
+      const createdStream = await Stream.create(newStream);
+      return res.send(createdStream);
+    } catch (error) {
+      return res.status(500).send(error);
+    }
+  },
+  async getById(req, res) {
+    try {
+      const stream = await Stream.findByPk(req.params.id);
+      if (stream) {
+        return res.status(200).send(stream);
+      } else {
+        return res.status(404).send({ error: 'Stream with this id was not found' });
+      }
+    } catch (error) {
+      return res.status(400).send(error);
+    }
+  },
+  async updateOne(req, res) {
+    try {
+      const stream = await Stream.findByPk(req.params.id);
+      if (stream) {
+        const updatedStream = await stream.update({
+          streamName: req.body.name || stream.name,
+        });
```

Comment:
Put не работает. На фронте мы получаем данные из include: [Stream] через student.Stream.name, поэтому пока лучше оставить просто req.body
```suggestion
        const updatedStream = await stream.update(req.body);
```